src/main/java/application/rest/v1/configmaps/SectionConfigMapProcessor.java

```diff
@@ -86,33 +85,29 @@ public JsonObject getConfigMap(ApiClient client, JsonObject component) {
     } 
          
 
-    private JsonObject getConfigMap(ApiClient client, String namespace, String configMapName) {
+    private JsonObject getConfigMap(ApiClient client, String configMapName) {
         // Return the map from the local cache if it's been previously loaded.
-        final boolean isGlobalNS = GLOBAL_NAMESPACE.equals(namespace);
-        if (isGlobalNS && kappnavNSMapCache.containsKey(configMapName)) {
+        if (kappnavNSMapCache.containsKey(configMapName)) {
             return kappnavNSMapCache.get(configMapName);
         }
         try {
             CoreV1Api api = new CoreV1Api();
             api.setApiClient(client);
-
-            V1ConfigMap map = api.readNamespacedConfigMap(configMapName, namespace, null, null, null);
+            
+            V1ConfigMap map = api.readNamespacedConfigMap(configMapName, KAppNavConfig.getkAppNavNamespace(), null, null, null);
```

Comment:
use  GLOBAL_NAMESPACE  (final variable at top of file)  instead of KAppNavConfig.getkAppNavNamespace()